manifests/params.pp

```diff
@@ -20,6 +20,25 @@
   $package_name       = 'vault'
   $package_ensure     = 'installed'
 
+  $checksum_type       = 'sha256'
+  $checksums           = {
```

Comment:
it should just output the checksums as puppet hash, so it can be pasted in the params file?